fe/src/main.js

```diff
@@ -1,11 +1,30 @@
-import Vue from 'vue'
+import Vue from 'vue';
+import Router from 'vue-router';
 import ElementUI from 'element-ui';
 import 'element-ui/lib/theme-chalk/index.css';
-import App from './App.vue'
+import App from './App.vue';
+import Login from './components/Login.vue';
+import Home from './components/Home.vue';
+import Register from './components/Register.vue'
 
 Vue.config.productionTip = false
 
+Vue.use(Router)
 Vue.use(ElementUI)
+
+const router = new Router({
+
+  routes: [
+    
+    {path: '/', component: Home},
+    {path: '/login', name:'Login', component:Login},
+    {path: '/register', component:Register},
+    
```

Comment:
why all these newlines, get rid of them

Reply:
sorry man I will try to keep it cleaner :)